index.html

```diff
@@ -122,288 +121,83 @@ <h2 class="item2">Multi-Post Stories Gain+Glory</h2>
-      <p class="sub-head">Skills</p>
-     <ul class="flex3">
-     <li class="lang">Database Management</li>
-     <li class="lang">Version Control</li>
-     </ul>
-     <ul class="flex4">
-       <li class="lang">CLI</li>
-       <li class="lang">Web development</li>
-       <li class="lang">API Design</li>
-     </ul>
-   </div>
-    </div>
-    <div class="flex5">
+            <p class="sub-head">Languages</p>
+             <ul class="flex2">
+              <li class="lang">JavaScript</li>
+              <li class="lang">Ruby</li>
+              <li class="lang">HTML</li>
+              <li class="lang">CSS</li>
+             </ul>
+       </div>
+       <div class="flex1">
+         <img id="icon" src="./features/images/Icon.png">
+          <p class="sub-head">Frameworks</p>
+            <ul class="flex3">
+             <li class="lang">React.js</li>
+             <li class="lang">Ruby on Rails</li>
+             <li class="lang">RSpec</li>
+           </ul>
+            <ul class="flex4">
+              <li class="lang">Capybara</li>
+              <li class="lang">Selenium</li>
+           </ul>
+        </div>
+       <div class="flex1">
+         <img id="icon" src="./features/images/ico.png">
+          <p class="sub-head">Skills</p>
+           <ul class="flex3">
+            <li class="lang">Database Management</li>
+            <li class="lang">Version Control</li>
+           </ul>
+          <ul class="flex4">
+           <li class="lang">CLI</li>
+           <li class="lang">Web development</li>
+           <li class="lang">API Design</li>
+         </ul>
+       </div>
+     </div>
+      <div class="flex5">
         <p id="bio">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
        </div>
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+    </section>
+  </main>
+   <footer>
+    <section>
+      <div id="vector">
+     <form action="https://formspree.io/f/xoqzonzv" method="post" class="form">
+       <div class="flex6">
+         <input type="text" name="name" placeholder="Full name" maxlength="30" class="text-placeholder" required>
+       </div>
+        <div class="flex6">
+          <input type="email" name="email" placeholder="Email address" class="text-placeholder" required>
+        </div>
+        <div class="flex7">
+         <textarea name="message" placeholder="Hey I’d like to connect to..." maxlength="500" class="style-placeholder"></textarea>
+         <input type="submit" name="submit" class="touch" value="Get in touch">
```

Comment:
- [ Required ] I think it would be better if you implemented the button interactions as requested by the previous reviewer, currently, when I hover or click the `Get in touch` button the styling of the button doesn't change. Please see the color pallete below for the styles you need to implement for the different interactions such as `hover`, and `pressed`

## Check the green button style changes
![](https://user-images.githubusercontent.com/83042742/244004080-e5b80f7c-d002-47f5-b029-879d3dca2786.png)